src/redux/books/Books.js

```diff
@@ -0,0 +1,33 @@
+// Actions
+const Addbook = 'ADD_BOOK';
+const Removebook = 'REMOVE_BOOK';
+
+// intial State
+const initialState = [];
+
+// Reducer
+const booksReducer = (state = initialState, action) => {
+  switch (action.type) {
+    case Addbook:
+      return [...state, action.payload];
+    case Removebook:
+      return state.filter((book) => book.id !== action.payload);
+    default:
+      return state;
+  }
+};
+
+// Action Creators
+const addBook = (book) => ({
+  type: Addbook,
+  payload: book,
+});
+
+const removeBook = (book) => ({
+  type: Removebook,
+  payload: book,
+});
+
+// Export
+export { addBook, removeBook };
+export default booksReducer;
```

Comment:
- [ ] Your books actions and reducers should be configured using `createSlice` from redux toolkit. Kindly refactor this code using `createSlice` instead. You can consider looking at [this article](https://www.softkraft.co/how-to-setup-slices-with-redux-toolkit/) for a general overview of the required setup.